docs/operators-and-hooks-ref.rst

```diff
@@ -147,10 +147,91 @@ Software operators and hooks
 These integrations allow you to perform various operations within software developed by Apache Software
 Foundation.
 
-.. operators-hooks-ref::
-   :tags: apache
-   :header-separator: "
+.. list-table::
```

Comment:
Please don't do this. This table is now generated automatically from provider.yaml, but you have to assign the tags.